### ISSUE TITLE
Launching analysis is impossible

### ISSUE THREAD

**Issue by hackdna** [parklab/refinery-platform#792]
Commit 6eec78c

### Steps to reproduce
1. Import the RFC test data set and test workflows
1. Open the data set and click on the Analyze tab
1. Select a file, a test workflow and Current Selection as Input Dataset
1. Click the Launch Analysis button

### Observed results
Nothing happens

### Expected results
Analysis is launched

### Notes
No relevant successful or failed requests in the web server log or celery logs. The following error is displayed in the browser console when the Analyze tab is opened:
```
Error: [$injector:unpr] http://errors.angularjs.org/1.4.8/$injector/unpr?p0=%24uibModalProvider%20%3C-%20%24uibModal%20%3C-%20rpAnalysisLaunchModalDirective
    at Error (native)
    at http://192.168.50.50:8000/static/vendor/angular/angular.min.js:6:416
    at http://192.168.50.50:8000/static/vendor/angular/angular.min.js:41:121
    at Object.d [as get] (http://192.168.50.50:8000/static/vendor/angular/angular.min.js:39:92)
    at http://192.168.50.50:8000/static/vendor/angular/angular.min.js:41:195
    at d (http://192.168.50.50:8000/static/vendor/angular/angular.min.js:39:92)
    at Object.e [as invoke] (http://192.168.50.50:8000/static/vendor/angular/angular.min.js:39:362)
    at http://192.168.50.50:8000/static/vendor/angular/angular.min.js:50:180
    at n (http://192.168.50.50:8000/static/vendor/angular/angular.min.js:7:333)
    at Object.<anonymous> (http://192.168.50.50:8000/static/vendor/angular/angular.min.js:50:147)
```

**Comment by jkmarx** [comment]
Double check your bower.json to ensure you have the correct version of angular-bootstrap 0.14.3

**Comment by hackdna** [comment]
Fixed in 1630bd7c
